sklearn/base.py

```diff
@@ -747,6 +748,34 @@ def fit_predict(self, X, y=None):
         return self.fit(X).predict(X)
 
 
+class OneToOneMixin:
```

Comment:
Since this is specific to transformers, shall we be more specific `OneToOneTransformerMixin`

Reply:
I would rather use `OneToOneFeatureMixin` to make it explicit that it's about feature mapping. In this case, it's implicit that this only makes sense for transformers.